util/awsservice/cloudwatchlogs.go

```diff
@@ -253,7 +253,7 @@ func AssertNoDuplicateLogs() LogEventsValidator {
 				byTimestamp[timestamp] = messages
 			}
 			_, ok = messages[message]
-			if !ok {
+			if ok {
```

Comment:
This seems like we are just skipping the test?

Reply:
The test checks to see if there are duplicate log events by creating a map of the timestamp to messages. If it finds a message within the same timestamp that's been previously set in this map, then it'll return this error, which means the test should fail.